Get rid of long-long feature. This breaks on AppleClang compilers.
---
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 40a7ab0..c84f0f7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -26,9 +26,9 @@ else()
 endif()
 
 # Enable C++11 long long for compilers that are capable of it
-if(NOT ${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} STRLESS 3.1)
-	target_compile_features(pugixml PUBLIC cxx_long_long_type)
-endif()
+# if(NOT ${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} STRLESS 3.1)
+# 	target_compile_features(pugixml PUBLIC cxx_long_long_type)
+# endif()
 
 set_target_properties(pugixml PROPERTIES VERSION 1.7 SOVERSION 1)
 
